### ISSUE TITLE
A recent update removes "=" in $route.query

### ISSUE THREAD

**Issue by joffreyBerrier** [nuxt/nuxt.js#8603]
### Versions

- nuxt: 2.14.12
- node: 15.4.0

### Reproduction

I use algolia on a nuxt application, and I push the currentFilter in the url:

```javascript
this.$router.push({ query: { numericFilters: 'capacity >= 5' } })
```

The url is then : http://localhost:3000/fr/recherche?numericFilters=capacity%20%3E%3D%205
On refresh, server-side with asyncData I get the query with route.query
- in v2.14.10 the decoded result was capacity >= 5
- In your current version the result is capacity >

On url decode https://www.freeformatter.com/url-encoder.html#ad-output
The result is capacity >= 5

The cause might come from your HotFix **#8494** named _Fix issues with URL handling (resolves #8497, #8493, #8458, #8457)_

<details open>
<summary>Additional Details</summary>
<br>

</details>

### Steps to reproduce
Steps to reproduce

The **$route.query** return `{ "numericFilters": "capacity >= 5" }`
The **querySsr** return `{ "numericFilters": "capacity >" }`

```javascript
<template>
  <div>
    <button @click="addQuery">Add query on url</button>
    <p>
      Query on clientSide : <strong>{{ $route.query }}</strong>
    </p>
    <p>
      Query on ssr : <strong>{{ querySsr }}</strong>
    </p>
  </div>
</template>

<script>
export default {
  asyncData({ route }) {
    // eslint-disable-next-line no-console
    console.log(route.query)

    return {
      querySsr: route.query,
    }
  },
  data() {
    return {
      isOpen: false,
    }
  },
  methods: {
    addQuery() {
      this.$router.push({ query: { numericFilters: 'capacity >= 5' } })
    },
  },
}
</script>

<style>
div {
  height: 100vh;
  width: 100vw;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
}
button,
p {
  margin: 0 auto 15px;
}
</style>
```

<img width="694" alt="Capture d’écran 2021-01-07 à 14 33 17" src="https://user-images.githubusercontent.com/26119557/103898498-4d5bbf00-50f5-11eb-85e3-b7031981f570.png">

### What is Expected?

Return the result of the UrlDecode => `{ "numericFilters": "capacity >= 5" }` like before

**Comment by pi0** [comment]
Hi @joffreyBerrier. Issue with parsing `=`  in query values resolved in ufo@0.5.3. Would you please try upgrading with `yarn upgrade nuxt` or `npm up nuxt` and try again? (locally testing, issue seems fixed so closing issue but please ping if not to investigate)

**Comment by Djules** [comment]
Just for the record, it fixed issue for me with Prismic preview mode in `@nuxtsj/prismic` module (the "websitePreviewId=" part from the `token` URL parameter was lost when converted to `query` parameter in nuxt `context`.
